src/aws/builder.rs

```diff
@@ -943,7 +943,7 @@ impl AmazonS3Builder {
         } else if self.access_key_id.is_some() || self.secret_access_key.is_some() {
             match (self.access_key_id, self.secret_access_key, self.token) {
                 (Some(key_id), Some(secret_key), token) => {
-                    info!("Using Static credential provider");
+                    debug!("Using Static credential provider");
```

Comment:
Given these messages are just static strings, I agree that debug is a more appropriate level of logging in my opinion